cardano-node/src/Cardano/Node/CLI.hs

```diff
@@ -4,29 +4,26 @@
 {-# LANGUAGE GADTs                #-}
 {-# LANGUAGE NamedFieldPuns       #-}
 {-# LANGUAGE OverloadedStrings    #-}
+{-# LANGUAGE RankNTypes           #-}
 
 {-# OPTIONS_GHC -Wno-all-missed-specialisations #-}
 
 module Cardano.Node.CLI (
   -- * Untyped/typed protocol boundary
     Protocol(..)
   , SomeProtocol(..)
-  , ViewMode(..)
   , fromProtocol
+  , parseProtocol
```

Comment:
`parseProtocol` is used in multiple places. This belongs in `Cardano.Node.Parsers`. Likewise for other parsers used in multiple places.

Reply:
Maybe then we also want to move `Protocol` -- to a new `Cardano.Node.Types` module, perhaps?

Reply:
Moved `Protocol` and `parseProtocol` into `Cardano.Node.Types`.

Reply:
> Maybe then we also want to move `Protocol` -- to a new `Cardano.Node.Types` module, perhaps, or even, `Cardano.Node.Protocol`?

https://github.com/input-output-hk/cardano-node/pull/143